tests/src/test/scala/tests.scala

```diff
@@ -27,7 +27,8 @@ class tests extends WordSpec with Matchers {
       import org.ensime.pcplod._
       withMrPlod("pcplodtest.scala") { pc =>
         pc.typeAtPoint('result) shouldBe Some("Option[Int]")
-        pc.typeAtPoint('test) shouldBe Some("(n: Int)freestyle.FreeS[F,Int]")
+        // Disabled test: the name of the parameter of `FreeS` is no longer known.
+        //pc.typeAtPoint('test) shouldBe Some("(n: Int)freestyle.FreeS[F,Int]")
```

Comment:
If the test is left activated, it fails with an error message such as the following one:
```scala
[info] - should generate code that works in the presentation compiler *** FAILED ***
[info]   Some("(n: Int)cats.free.Free[[β$0$]cats.free.FreeApplicative[F,β$0$],Int]") 
was not equal to Some("(n: Int)freestyle.FreeS[_,Int]") (tests.scala:31)
```
It seems that the aliases for `OpSeq` and `OpPar` are now being expanded. I have made a few small changes to the test, so now we may keep it.
Now, thep